src/components/WorkOrders/CloseWorkOrder.js

```diff
@@ -70,7 +101,7 @@ const CloseWorkOrder = ({ reference }) => {
         <>
           {CloseWorkOrderFormPage && (
             <CloseWorkOrderForm
-              reference={reference}
+              reference={workOrder?.reference}
```

Comment:
If we aren't using the `workOrder` object anywhere else probably could just set the reference directly from the `await getWorkOrder(reference)` call to avoid having to use a safe operator here but not significant at all

Reply:
I've reworked this PR to avoid using this operator.